src/runner.ts

```diff
@@ -1,39 +1,49 @@
 import { checkDifferences } from './checkDifferences';
 import { collect } from './collect';
 import { createShots } from './createShots';
-import { createShotsFolders, getEventData, log } from './utils';
+import { createShotsFolders, getEventData, log, isUpdateMode } from './utils';
 import { config, configure } from './config';
 import { sendResultToAPI } from './upload';
 import { sendInitToAPI } from './sendInit';
 
 (async () => {
   await configure();
+  const isGenerateOnlyMode = config.generateOnly;
   try {
-    if (config.setPendingStatusCheck) {
+    if (config.setPendingStatusCheck && !isGenerateOnlyMode) {
```

Comment:
how about using `config.generateOnly` here as well, as it is that way below too and gives more context?

Reply:
[47f6701](https://github.com/lost-pixel/lost-pixel-action/pull/16/commits/47f67018fc9a171b06c4d36d057746b004b85507)